src/coreclr/tools/aot/ILCompiler.ReadyToRun/Compiler/ReadyToRunMetadataFieldLayoutAlgorithm.cs

```diff
@@ -196,9 +196,9 @@ protected override ModuleFieldLayout CreateValueFromKey(EcmaModule module)
                         }
```

Comment:
I would expect that the call to `MarshalUtils.IsBlittableType` is going disappear from this file as part of this change. Is it possible?

Reply:
We'd need to get rid of the explicit layout size compat quirk in both CoreCLR and here, which I'm okay doing if we don't have concerns about removing it.

Reply:
I would love to get rid of this quirk if possible.

Reply:
I'll try to remove it then.